string epc
int64 ts
int16 antenna_port
float32 RSSI